packages/office-ui-fabric-react/src/common/sharedIsConformant.ts

```diff
@@ -0,0 +1,11 @@
+import { isConformant, IsConformantOptions } from '@fluentui/react-conformance';
+import { Ref } from '@fluentui/react-component-ref';
+
+export function sharedIsConformant(testInfo: IsConformantOptions) {
+  const defaultOptions = {
+    disabledTests: ['has-docblock'],
+    helperComponents: [Ref],
```

Comment:
OUFR and react-next don’t use Ref. This should be either not provided or just have FocusZone.